cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:3000",
+  "videoUploadOnPasses": false
```

Comment:
We only want videos when tests are failing.

Reply:
Agree! 😄 

I think the idea is that you can see on a Cypress Dashboard when things go wrong, and then see the video of what went wrong, so you can fix it?

Reply:
Exactly, so you don't need to rerun it locally to reproduce the error.

Reply:
That's nice! 😄